stubs/Modules/Base/shell.sh

```diff
@@ -7,6 +7,7 @@ npm install swiper@9.3.2
 composer require dejury/gptfaker --dev
 composer require awcodes/filament-curator
 composer require solution-forest/filament-tree
+composer require motivo/filament-title-with-slug
```

Comment:
Kunnen we deze code niet in deze repo opnemen? Ik vind het zo jammer dat de prefab nu uit 2 packages bestaat die beide van ons zijn.

Reply:
Was ook mijn eerste ingeving, maar denk dat forken praktischer is. Dan kunnen wij namelijk meeliften op werk van anderen. Als er een bug is of bijv. een filament/laravel upgrade dan kunnen wij het werk van anderen makkelijk in onze fork overnemen. Voor bestaande projecten hoeven wij dan ook alleen maar een composer update te doen ipv bestanden 1 voor 1 overnemen

Reply:
Maar waarom hebben we dan nog een fork nodig? Dan kan je toch gewoon de source gebruiken? Ik zie niet helemaal de meer waarde van onze eigen fork dan.